src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -59,12 +67,35 @@ private static object RegExSetup(PowerFxConfig config)
             return null;
         }
 
+        private static void AddResourceManager(RuntimeConfig runtimeConfig)
+        {
+            runtimeConfig.AddService<IResourceManager>(new DefaultResourceManager());
+        }
+
+        private static object BlobSetup(PowerFxConfig config, SymbolTable symbolTable)
+        {
+            config.AddBlobTestFunctions();
+            config.EnableSetFunction();
+
+            return new List<ISymbolSlot>()
+            {
+                symbolTable.AddVariable("blob", FormulaType.Blob, true, "blob"),

```

Comment:
>true [](http://example.com/codeflow?start=66&length=4)

Blobs shouldn't be mutable (from the point of view of Power Fx). Nor should strings (line below)

Reply:
They aren't mutable & can have multiple forms